misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueModule.kt

```diff
@@ -17,6 +17,13 @@ import java.util.concurrent.Executors
 
 /** [AwsSqsJobQueueModule] installs job queue support provided by SQS */
 class AwsSqsJobQueueModule(private val config: AwsSqsJobQueueConfig) : KAbstractModule() {
+  override fun configure() {
+    multibind<Service>().to<SqsJobConsumer>()
+    install(AwsSqsJobQueueCommonModule(config))
+  }
+}
+
+internal class AwsSqsJobQueueCommonModule(private val config: AwsSqsJobQueueConfig) : KAbstractModule() {
```

Comment:
what's the benefit of splitting this out? Doesn't seem used elsewhere

Reply:
Ah, yeah this was me trying to add tests by splitting apart the service. Will revert